gno.land/cmd/gnoland/start.go

```diff
@@ -374,17 +374,14 @@ func initializeLogger(io io.WriteCloser, logLevel, logFormat string) (*zap.Logge
 }
 
 func generateGenesisFile(genesisFile string, pk crypto.PubKey, c *startCfg) error {
+	bp := bft.DefaultBlockParams()
+	bp.InitialGasPriceDenom = "ugnot"
+
 	gen := &bft.GenesisDoc{}
 	gen.GenesisTime = time.Now()
 	gen.ChainID = c.chainID
 	gen.ConsensusParams = abci.ConsensusParams{
-		Block: &abci.BlockParams{
-			// TODO: update limits.
-			MaxTxBytes:   1_000_000,   // 1MB,
-			MaxDataBytes: 2_000_000,   // 2MB,
-			MaxGas:       100_000_000, // 100M gas
-			TimeIotaMS:   100,         // 100ms
-		},
+		Block: bp,
```

Comment:
please revert, this should be set per application, in this case as before in cmd/gnoland/start.go for the gnoland application.